modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -882,7 +882,15 @@ List configured mounts.
 
 [source,console]
 ----
-files_external:list [--show-password] [--full] [-a|--all] [-s|--short] [--] [<user_id>]
+files_external:list
```

Comment:
Sorry for any confusion caused, but I only meant to have linebreaks for the code example, not for anything else.

Reply:
We have this look and feel also on other locations where the list or length of options and arguments is long and therefore broke it into multi lines. Example see https://doc.owncloud.org/server/10.3/admin_manual/configuration/server/occ_command.html#files_externalcreate

Reply:
Fair enough.